presto-docs/src/main/sphinx/connector/hive-s3.rst

```diff
@@ -158,6 +158,15 @@ For example, the mapping list might have URL prefix ``s3://abc/xyz/`` followed b
 ``s3://abc/`` to allow different configuration for a specific path within a bucket
 than for other paths within the bucket. You can set default configuration by not
 including any match criteria for the last entry in the list.
+In addition to rules above the default mapping could contain
+``"useClusterDefault": "true"`` configuration property
+to use the default cluster role as a fallback option.
+``useClusterDefault`` could not be used together with other configuration properties
+(``accessKey``, ``secretKey``, ``iamRole``, ``allowedIamRoles``).
+In case there is no any mapping entry that fits requirements and no default
```

Comment:
If no mapping entry matches and no default is configured, an error ``AccessDeniedException: Access Denied: No matching S3 security mapping`` is returned.

Reply:
done